### PR TITLE
TC 16.1.1_05 Men/Bottoms/Shopping options/Category Verify that the quantity of available items is visible

### PR DESCRIPTION
**Pull request by VeraHarbachova**
https://trello.com/c/3NOwcdPh/737-tc-161105-men-bottoms-shopping-options-category-verify-that-the-quantity-of-available-items-is-visible

### CODE REVIEW

File: tests/test_page_objects/menBottomsPage.spec.js [hunk 1]
```diff
@@ -84,5 +84,28 @@ import { BASE_URL, MEN_BOTTOMS_PAGE_END_POINT, LIST_CATEGORY_MEN_BOTTOMS, ID_PAR
...
+
+      await menBottomsPage.hoverMenBottomsCategory();
+      await menBottomsPage.clickMenBottomsCategory();
+      await menBottomsPage.waitForTimeout(3000);
+
+      await expect(menBottomsPage.locators.getMenBottomsCategoryListOfItemsLocator().first()).toBeVisible();
+  
+      const receivedElements = (await menBottomsPage.locators.getMenBottomsCategoryListOfItemsLocator().allInnerTexts()).map(item => item.replace('\n' , ' ')); 
+  
+      console.log(receivedElements);
```
Comment (IrenaKh95) [inline tests/test_page_objects/menBottomsPage.spec.js:106]: вы забыли удалить console.log

Comment (VeraHarbachova) [reply tests/test_page_objects/menBottomsPage.spec.js:106]: спасибо)

**Review by IrenaKh95 — DISMISSED** [review]

File: tests/test_page_objects/menBottomsPage.spec.js [hunk 2]
```diff
@@ -84,5 +84,26 @@ import { BASE_URL, MEN_BOTTOMS_PAGE_END_POINT, LIST_CATEGORY_MEN_BOTTOMS, ID_PAR
       await expect(menBottomsPage.locators.getMenBottomsFilterList()).toHaveClass(/active/);
   
     })
+
+    test("verify that the quantity of available items is visible", async ({ page }) => {
+      const homePage = new HomePage(page);
+      const menBottomsPage = new MenBottomsPage(page);
```
Comment (verafes) [inline tests/test_page_objects/menBottomsPage.spec.js:90]: please do not create a new page, as this is required on the project (pls review last Maria lecture on this). You may combine lines 90 and 92 like this:
const menBottomsPage = await homePage.hoverMenLink();
and pls change 93 line: 
await menBottomsPage.clickMenBottomsLink();

Comment (VeraHarbachova) [reply tests/test_page_objects/menBottomsPage.spec.js:90]: Unfortunately, this option will not work. Assigning the hover method to a constant will not lead to a new page. Using the created page in the following steps does not work.

Comment (VeraHarbachova) [reply tests/test_page_objects/menBottomsPage.spec.js]: Changed the constants to: 
const homePage = new HomePage(page);
 await homePage.hoverMenLink();
 const menBottomsPage = await homePage.clickMenBottomsLink();

**Review by mlimau — APPROVED** [review]

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**